django_auth_lti/static/django_auth_lti/js/resource_link_id.js

```diff
@@ -0,0 +1,12 @@
+window.globals.append_resource_link_id = function(url){
+    if (!url.match(/resource_link_id/)) {
+        if (!url.match(/\?/g)) {
+            url += '?';
+        }
+        return url + '&resource_link_id=' + window.globals.RESOURCE_LINK_ID;
+    }
+};
+
+$(document).ajaxSend(function(event, jqxhr, settings){
```

Comment:
Yeah, the only problem here is you need to revert to manually calling window.globals.append_resource_link_id() if you aren't using JQuery to send Ajax requests, which we won't be if we are using Angular.